src/user/dto/create-user-dto.ts

```diff
@@ -0,0 +1,6 @@
+export class CreateUserDto {
+    username: string;
+    password: string;
+    email: string;
```

Comment:
you should add `@IsEmail()` above the email propertry to help `class-validator` validate the email's format.